resources/js/util/address.ts

```diff
@@ -2,14 +2,15 @@ import { decodeAddress, encodeAddress } from '@polkadot/util-crypto';
 import { u8aToHex, isHex, hexToU8a } from '@polkadot/util';
 import { useAppStore } from '~/store';
 
+const CANARY_NETWORK_ID = 9030;
+const ENJIN_NETWORK_ID = 1110;
```

Comment:
Just so we can keep consistent with the other applications, we now use for the network consts
ENJIN_MATRIXCHAIN
CANARY_MATRIXCHAIN
ENJIN_RELAYCHAIN
CANARY_RELAYCHAIN